app/package.json

```diff
@@ -1,6 +1,16 @@
 {
   "name": "radicle-upstream",
-  "version": "0.0.1",
+  "version": "0.0.2",
+  "author": {
+    "name": "Monadic",
```

Comment:
Ah yeah, dropped that action item from a few weeks back: we've been using "team@monadic.xyz" or even "radicle@monadic.xyz" (which is the public mailing list). I think we actually should have something like "dev@radicle.xyz", even though this may not save us from stating Monadic GmbH's copyright somewhere ... :S

Reply:
We’re using `radicle@monadic.xyz` as the author. I think `dev@` would be ideal.

Reply:
I like the idea of using "Radicle" as author and "dev@radicle.xyz" for the email, but will wait for @cloudhead's input.

Reply:
+1 @rudolfs

Reply:
(We won't receive those emails though, but is that a problem?)

Reply:
Hm, in that case I'd rather use an email that is working (radicle@monadic.xyz), in the off-chance that somebody thinks of reaching out to us.

Reply:
Ok, since in the meantime dev@radicle.xyz was created, I'm using that.